apps/redi-talent-pool/src/pages/app/browse/BrowseJobseeker.tsx

```diff
@@ -401,9 +415,15 @@ export function BrowseJobseeker() {
                 <JobListingCard
                   key={jobListing.id}
                   jobListing={jobListing}
-                  toggleFavorite={handleFavoriteJobListing}
-                  isFavorite={isFavorite}
                   linkTo={`/app/job-listing/${jobListing.id}`}
+                  renderCTA={() => renderFavoriteCTA(jobListing.id, isFavorite)}
+                  timestamp={formatDistance(
+                    new Date(jobListing.updatedAt),
+                    new Date(),
+                    {
+                      addSuffix: true,
```

Comment:
Also, I wonder if we should round the timestamp values `n hours/minutes ago` and `in n hours/minutes` to `today`, for example. But it's a product decision, we can discuss it with @astkhikatredi and fix it later. 
<img width="860" alt="image" src="https://github.com/talent-connect/connect/assets/51786805/7e099385-4793-495d-ae03-b5e323ea7a99">

We would also need to fix the truncation + padding issues for skills tags:
<img width="527" alt="image" src="https://github.com/talent-connect/connect/assets/51786805/09970a11-63f2-4676-b43f-a03150a5500b">

Reply:
Let's use the [standard `Intl.RelativeTimeFormat` here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat/format)? Browser implementation should probably do a more reliable job than `date-fns`.

Reply:
I will create myself a follow-up ticket on this one, to convert moment and date-fns functions to Intl.